crates/bevy_derive/src/bevy_main.rs

```diff
@@ -10,10 +10,12 @@ pub fn bevy_main(_attr: TokenStream, item: TokenStream) -> TokenStream {
     );
 
     TokenStream::from(quote! {
-        // use ndk-glue macro to create an activity: https://github.com/rust-windowing/android-ndk-rs/tree/master/ndk-macro
+        
         #[cfg(target_os = "android")]
-        #[cfg_attr(target_os = "android", bevy::ndk_glue::main(backtrace = "on", ndk_glue = "bevy::ndk_glue"))]
-        fn android_main() {
+        fn android_main( android_app: bevy::android::AndroidApp) {
+            // Store android app in a global variable
+            bevy::android::ANDROID_APP.set(android_app).unwrap();
```

Comment:
Same comment as above really, about trying to avoid treating this as global/static state ideally